server-src/db_mgr.js

```diff
@@ -99,22 +99,22 @@ async function initDB(config, useSocket=false) {
    * @param {string} tableName
    * @param {Array<string>} dataKeys
```

Comment:
Maybe add descriptions to explain the difference/usage between `data.*` and `query.*` params.

Reply:
Done